benchbuild/experiment.py

```diff
@@ -124,49 +131,56 @@
         return type(self).SCHEMA
 
     @schema.validator
-    def validate_schema(self, _, new_schema):
+    def validate_schema(self, _: tp.Any, new_schema) -> bool:
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5694346)